static/app/views/explore/charts/index.tsx

```diff
@@ -189,6 +220,31 @@ export function ExploreCharts({query}: ExploreChartsProps) {
                       options={intervalOptions}
                     />
                   </Tooltip>
+                  <Feature features="organizations:alerts-eap">
+                    <Tooltip
+                      title={
+                        singleProject
+                          ? t('Create an alert for this chart')
+                          : t(
+                              'Cannot create an alert when multiple projects are selected'
+                            )
+                      }
+                    >
+                      <DropdownMenu
+                        triggerProps={{
+                          'aria-label': t('Chart Actions'),
```

Comment:
I feel like this should be labelled something like "Create Alert from Query" or something. Even just "Create Alert"?